web/src/components/molecules/Member/MemberCreationModal/index.tsx

```diff
@@ -72,19 +72,21 @@ const MemberCreationModal: React.FC<Props> = ({
     form
       .validateFields()
       .then(async () => {
-        if (searchedUser?.id) await onSubmit?.();
+        if (searchedUser?.id) await onSubmit?.([searchedUser.id]);
+        onClose?.(true);
         form.resetFields();
```

Comment:
Does this modal actually create a member, or add a member to a workspace? If it is the latter, the name for this component is confusing and maybe `MemberAddModal` is more clear.